src/one_dragon_qt/view/setting/setting_instance_interface.py

```
@@ -297,39 +170,33 @@ def _get_instanceSwitch_group(self) -> QWidget:
         self.custom_win_title_input.setFixedWidth(214)
-        self.custom_win_title_input.editingFinished.connect(
-            self._update_custom_win_title
-        )
+        self.custom_win_title_input.editingFinished.connect(self._update_custom_win_title)
         self.custom_win_title_opt = PasswordSwitchSettingCard(
             icon=FluentIcon.FIT_PAGE,
-            title="自定义窗口标题",
+            title='自定义窗口标题',
             extra_btn=self.custom_win_title_input,
-            password_hash=base64.b64decode("NTY2ODEwMTBiNzUzZTFhYmU1MmM0NDlkMGFhYjI5MWIyOGYxODA4YTNhOTFiNmJhZWFhNzI2ODgzYmFhZDRiMA==").decode('utf-8'),
+            password_hash='56681010b753e1abe52c449d0aab291b28f1808a3a91b6baeaa726883baad4b0',
```

Ruff (0.13.2)

Possible hardcoded password assigned to argument: "password_hash"

(S106)